components/kit/components/commerce/shopping/BookingCard.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from "react";
+
+const BookingCard: FC = () => {
+  return (
+    <div className="lg:flex  max-w-2xl mx-auto overflow-hidden rounded-lg m-24  shadow-lg">
+      <img
```

Comment:
Images must be serve locally, please download this image on the project. 
Then the image must be resized, compressed and optimised so that it is as light as possible (here around 30 ko). 
You can do it with this tools : https://ezgif.com/resize for example.

Reply:
Can't you wrap that in the next image component if possible usually from version 10 I tend to use that in my projects